spec/features/admin_spec.rb

```diff
@@ -19,6 +19,7 @@
       within("tr#review_#{review.id}") do
         find('.approve').click
       end
+      sleep(3)
```

Comment:
Remove this.

Reply:
if we removed this, the spec would fail due to the delay of response.